### DISCUSSION TITLE
P.112 SECTION-048 リスト12.6 ログミドルウェアの実装例について

### DISCUSSION THREAD

**Discussion by litencatt** [budougumi0617/go_todo_app#33] [Ideas]
素晴らしい技術本の執筆ありがとうございます。微力ながら貢献できたらと思い今回連絡させていただきました:bow:

```go
// ...

func (r *rwWrapper) WriteHeader(statusCode int) {
	r.status = status
	r.rw.WriteHeader(statusCode)
}

// ...
```
は`r.status = statusCode`が正しそうでした。

**Comment by budougumi0617** [comment]
こちらこそご購入ありがとうございます！正誤表に追加させていただきました :pray:
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.4
